documents/policy/policy.tex

```diff
@@ -0,0 +1,168 @@
+\documentclass{scrreprt}
+\usepackage{listings}
+\usepackage{underscore}
+\usepackage[bookmarks=true]{hyperref}
+\usepackage[utf8]{inputenc}
+\usepackage[english]{babel}
+\usepackage{graphicx}
+\usepackage{xcolor}
+\usepackage{float}
+\hypersetup{
+    bookmarks=false,    % show bookmarks bar?
+    pdftitle={Team 2 Policies},    % title
+    pdfauthor={CMPT371 Team 2},                     % author
+    pdfsubject={Policy},                        % subject of the document
+    pdfkeywords={TeX, LaTeX, graphics, images, policy}, % list of keywords
+    colorlinks=true,       % false: boxed links; true: colored links
+    linkcolor=blue,       % color of internal links
+    citecolor=black,       % color of links to bibliography
+    filecolor=black,        % color of file links
+    urlcolor=purple,        % color of external links
+    linktoc=page            % only page is linked
+}%
+\def\myversion{1.0 }
+\date{}
+%\title{%
+
+%}
+\usepackage{hyperref}
+\begin{document}
+
+\graphicspath{ {images/} }
+
+\begin{flushright}
+    \rule{16cm}{5pt}\vskip1cm
+    \begin{bfseries}
+        \Huge{TEAM 2 POLICY}\\
+        \vspace{1.9cm}
+        for\\
+        \vspace{1.9cm}
+        CMPT371\\
+        \vspace{1.9cm}
+        \LARGE{Version \myversion approved}\\
+        \vspace{1.9cm}
+        Prepared by Team 2\\
+        \vspace{1.9cm}
+        University of Saskatchewan\\
+        \vspace{1.9cm}
+        \today\\
+    \end{bfseries}
+\end{flushright}
+
+\tableofcontents
+
+\chapter{Policy}
+
+\section{Members}
+
+\begin{table}[h!]
+\centering
+\begin{tabular}{ |p{2cm}||p{8cm}|  }
+    \hline
+    \multicolumn{2}{|c|}{Team Roles} \\
+    \hline
+    Member & Roles\\
+    \hline
+    Amanda & Project Lead\\
+    Mesa & Design Lead\\
+    Evan & Test Lead\\
+    Braunson & Build Master\\
+    Kevin & Risk Officer, Development Team, Test Team\\
+    Clinton & Test Team\\
+    Anurag & Development Team, Test Team\\
+    Eileen & Development Team\\
+    Camille & Development Team\\
+    \hline
+\end{tabular}
+\end{table}
+
+\section{Roles and Responsibilities}
+
+\subsection{Project Lead}
+
+The project lead will be responsible for organizing meetings,
+meeting with stakeholders, establishing priority, coordinating
+between accountable positions, and other administrative work. 
+Generally, the project lead will not directly code.
+In cases of disagreements the project lead has executive decision making power.
```

Comment:
Establishing priority of what? Should probably clarify that a bit. Also, I'm assuming "administrative work" includes emailing the stakeholders, taking meeting minutes, etc. (Just wondering how much detail we want with that).

Reply:
Adjusted the priority line.

And yea I note in this paragraph they are responsible for organizing meetings (agenda/minutes), and I mention meeting with stakeholders. Other administrative work could be emailing them right.